src/commissaire_http/handlers/hosts.py

```diff
@@ -152,12 +152,23 @@ def delete_host(message, bus):
     :returns: A jsonrpc structure.
     :rtype: dict
     """
-    # TODO: kick off service job to remove the host
     try:
```

Comment:
nit: Not required, but `address = message['params']['address']` here would save some typing below.